### ISSUE TITLE
How to change name of deployment to something other than "generic"

### ISSUE THREAD

**Issue by a2geek** [genesis-community/generic-genesis-kit#2]
I would like to use the generic kit in a couple of instances, but I do not see how to change the deployment name.

I have set the environment file to include:
```
params:
  name:         gdc-prod-cfcr
```

But the manifest itself still has `gdc-prod-generic` and a Genesis deploy fails with:
```
sing deployment 'gdc-prod-cfcr'

Expected manifest to specify deployment name 'gdc-prod-cfcr' but was 'gdc-prod-generic'

Exit code 1
```

Any suggestions?

**Comment by ramonskie** [comment]
i have no idea how your manifest looks.
but it seems to me that you need to parameterize the name in your manifest like `ops/base.yml`
and set:
```
name: (( grab params.name ))
````

**Comment by a2geek** [comment]
Ah! I think this is likely user error. 

What I missed is that `genesis init` creates a `.genesis/config` file. In there is the `deployment_type`. I had assumed that was the name of the kit and left it alone (because it was `generic`). 

I just retried with a fresh deployment and noticed that the init subcommand accepts an optional name. I tried that but couldn't find the name anywhere... until I looked in that config file. A `genesis deploy` confirmed that the deployment was `env-name` as expected instead of `env-generic`.

Assuming this is true, I think that it should be at least added to some setup documentation. I suspect that making it _required_ isn't possible due to the nature of Genesis itself.

Essentially the docs require some verbiage on how to setup the generic kit, including the required `name` component for `genesis init`:
```
genesis init --kit generic <your-name-here>
```
